src/platforms/uplay/platform.rs

```diff
@@ -297,4 +303,27 @@ impl GamesPlatform for UplayPlatform {
     fn code_name(&self) -> &str {
         "uplay"
     }
-}
\ No newline at end of file
+}
+
+#[cfg(test)]
+mod test {
+
+    use super::*;
+
+    #[test]
+    fn can_parse_configuration_file() {
+        let content = include_bytes!("testconfiguration");
+        let splits = get_file_splits(content);
+        assert_eq!(501, splits.len());
+    }
+
+   #[test]
+    fn can_parse_into_game_config() {
+        let content = include_bytes!("testconfiguration");
+        let splits = get_file_splits(content);
+        let games:Vec<_> = splits.iter().flat_map(|split| parse_game_config(split)).collect();
+        assert_eq!(2, games.len());
+        assert_eq!(Some("For Honor"),games.get(0).map(|h|h.shortcut_name));
+        assert_eq!(Some("WATCH_DOGS@ 2"),games.get(1).map(|h|h.shortcut_name));
```

Comment:
This work on my steam deck, here is a suggestion that should fix the test.
```suggestion
        assert_eq!(Some("WATCH_DOGS® 2"),games.get(1).map(|h|h.shortcut_name));
```